server/config/chart_config/crime.json

```diff
@@ -62,5 +62,69 @@
       "denominator": "Count_Person",
       "scaling": 100000
     }
+  },
+  {
+    "category": "Crime",
+    "titleId": "CHART_TITLE-Hate_Crime",
+    "title": "Hate Crime",
+    "description": "Total number of hate crime incidents",
+    "statsVars": ["Count_CriminalIncidents_IsHateCrime"],
+    "isOverview": true,
+    "relatedChart": {
+      "titleId": "CHART_TITLE-Hate_Crime_rate",
+      "title": "Hate crimes per 100,000 people",
+      "description": "Total number of hate crime incidents per 100,000 people",
+      "scale": true,
+      "denominator": "Count_Person",
+      "scaling": 100000
+    }
+  },
+  {
+    "category": "Crime",
+    "titleId": "CHART_TITLE-Hate_Crime_Gender",
+    "title": "Hate Crime motivated by gender",
+    "description": "Total number of hate crime incidents motivated by gender",
+    "statsVars": ["Count_CriminalIncidents_BiasMotivationGender_IsHateCrime"],
+    "isOverview": false
```

Comment:
please add relatedCharts for this and the remaining charts you've added

Reply:
I had skipped those because the numbers might seem too large if population is small. Added now.